spec/models/target_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe Target, type: :model do
+
+  describe  "#load_or_create" do
```

Comment:
You are testing the model. If you put #load_or_create it means that you are testing that method from the controller. In this case I think you are grouping validations, so you can call it that.